pages/send/Send.tsx

```diff
@@ -99,25 +142,20 @@ export function Send() {
               className="flex flex-col gap-2 flex-1"
             >
               <KeyboardIcon className="text-muted-foreground" />
-              <Text numberOfLines={1}>Manual</Text>
             </Button>
             <Button
+              onPress={paste}
               variant="secondary"
               className="flex flex-col gap-2 flex-1"
-              onPress={() => {
-                router.push("/send/address-book");
-              }}
             >
-              <BookUserIcon className="text-muted-foreground" />
-              <Text numberOfLines={1}>Contacts</Text>
+              <PasteIcon className="text-muted-foreground" />
             </Button>
             <Button
-              onPress={paste}
+              onPress={pickImage}
               variant="secondary"
               className="flex flex-col gap-2 flex-1"
             >
-              <PasteIcon className="text-muted-foreground" />
-              <Text numberOfLines={1}>Paste</Text>
+              <ImageIcon className="text-muted-foreground" />
```

Comment:
I think it's always good to have text for a button no? It is also not consistent with the receive screen and home screen currently

Reply:
Rene also thinks it looks good, resolving for now